src/lib.rs

```diff
@@ -3106,7 +3106,12 @@ impl Build {
             Some(t) => t,
             None => {
                 if target.contains("android") {
-                    name = format!("{}-{}", target.replace("armv7", "arm"), tool);
+                    let compiler = self.get_base_compiler()?;
+                    if compiler.family == ToolFamily::Clang {
```

Comment:
Hi, please let me check through major NDK releases to find the best way. In my understanding, when Clang is used, then it needs `llvm-` tools while for GCC they have `prefix-` tools. So IMO it's safe to assume. Besides, by checking command availability I'm a bit worried about resolving installed tools that's already in PATH.

Reply:
Since Android NDK r18b (released in 2018), GCC has been completely removed. So I think it can even be simplified to use `llvm-` only.

https://developer.android.com/ndk/downloads/revision_history

Reply:
I'd recommend to try running it using `std::process::Command`

Reply:
Should the order for testing command to be LLVM > GCC?

Reply:
> Should the order for testing command to be LLVM > GCC?

Yeah, LGTM

Reply:
Got it. I'll update the patch soon!

Reply:
Done!